### DISCUSSION TITLE
Custom user folders

### DISCUSSION THREAD

**Discussion by michalheger** [UniSharp/laravel-filemanager#1149] [General]
Hello, 
I'd like to ask, if there is any method of using existing folders for users (ex. project folders). Functions for that background are done, only issue is to tell the system the list of accessible folders for that user.

**Comment by streamtw** [comment]
It is possible, by these steps:
1. In `config/filesystems.php`, create a new entry in `disks`(`driver` should be `local`), setting the `root` to the path you want.
2. In `config/lfm.php`, change the value of `disk` to the disk you just created in `config/filesystems.php`.

**Reply by stevensu123** [reply]
Can Unisharp display the file manager with the "background" folder when the "background" button is selected, or the "author" folder when the "author" button is selected?
